extensions/sonos/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "sonos",
+  "title": "Sonos",
+  "description": "Control your Sonos system",
+  "icon": "sonos.png",
+  "author": "AntonNiklasson",
+  "categories": [
+    "Media",
+    "Fun",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "play-pause",
+      "title": "Play / Pause",
+      "subtitle": "Sonos",
+      "description": "Toggle the playback state of the active group",
+      "mode": "no-view",
+      "interval": "5s"
+    },
+    {
+      "name": "set-group",
+      "title": "Set Active Group",
+      "subtitle": "Sonos",
+      "description": "Pick one of the household groups to be the active group",
+      "mode": "view"
+    },
+    {
+      "name": "now-playing",
+      "title": "Now Playing",
+      "subtitle": "Sonos",
+      "description": "Show the currently playing media in the menu bar",
+      "mode": "menu-bar"
+    },
+    {
+      "name": "increase-volume",
+      "title": "Increase Volume",
+      "subtitle": "Sonos",
+      "disabledByDefault": true,
+      "description": "Increase volume in the active group",
+      "mode": "no-view"
+    },
+    {
+      "name": "decrease-volume",
+      "title": "Decrease Volume",
+      "subtitle": "Sonos",
+      "disabledByDefault": true,
```

Comment:
What was the reason for this change, @pernielsentikaer? If we handle the case of running these commands without a Sonos system present, perhaps all commands can be enabled by default? Or do you have some best-practice in mind around this?

Reply:
Replied back on your Slack DM 🙂

Reply:
I've disabled the skip commands by default as well 🙌